packages/client/hmi-client/src/components/widgets/tera-slider-panel.vue

```diff
@@ -6,7 +6,7 @@
 		:is-open="isOpen"
 	>
 		<template v-slot:content>
-			<header class="slider-header content">
+			<header class="slider-header content sticky">
 				<i
```

Comment:
What slider panel is being effected here? I can't see the difference with adding the class